core/src/apps/bitcoin/scripts_decred.py

```diff
@@ -141,3 +151,54 @@ def sstxcommitment_pkh(pkh: bytes, amount: int) -> bytes:
     write_uint64_le(w, amount)
     write_bytes_fixed(w, b"\x00\x58", 2)  # standard fee limits
     return w
+
+
+def output_script_p2pkh(pubkeyhash: bytes) -> bytearray:
+    s = utils.empty_bytearray(25)
+    scripts.write_output_script_p2pkh(s, pubkeyhash)
+    return s
+
+
+def output_script_p2sh(scripthash: bytes) -> bytearray:
+    # A9 14 <scripthash> 87
+    utils.ensure(len(scripthash) == 20)
+    s = bytearray(23)
+    s[0] = 0xA9  # OP_HASH_160
+    s[1] = 0x14  # pushing 20 bytes
+    s[2:22] = scripthash
+    s[22] = 0x87  # OP_EQUAL
+    return s
+
+
+def output_derive_script(tree: int, stakeType: int, addr: str, coin: CoinInfo) -> bytes:
+    from trezor.crypto import base58
+    from apps.common import address_type
```

Comment:
isort wants an empty line here:
```suggestion
    from trezor.crypto import base58

    from apps.common import address_type
```